proms-resources/frontend/src/main/frontend/src/proms/ToUDialog.jsx

```diff
@@ -122,7 +122,7 @@ function ToUDialog(props) {
     return null;
   }
 
-  if (tou && touAcceptedVersion && tou.version == touAcceptedVersion) {
+  if (tou && (touAcceptedVersion && tou.version == touAcceptedVersion || !tou.enabled) ) {
     onAccept && onAccept();
```

Comment:
Let's rename this too to match the new semantic:
```suggestion
    onCleared && onCleared();
```